tests/export/test_nemo_export.py

```diff
@@ -214,6 +214,8 @@ def run_trt_llm_inference(
             max_prompt_embedding_table_size=max_prompt_embedding_table_size,
             use_lora_plugin=use_lora_plugin,
             lora_target_modules=lora_target_modules,
+            max_num_tokens=int(max_input_token * max_batch_size * 0.2),
```

Comment:
Possible to add a comment in this line why `max_input_token * max_batch_size * 0.2`, e.g. what 0.2 stands for and how this is derived in general